building/tracks/new/add-first-exercise.md

```diff
@@ -25,3 +22,52 @@ The "Hello, World!" exercise has some special rules applied to it:
 - The stub file contains an almost-working implementation, but instead of "Hello, World!" it uses "Goodbye, Mars!"
 - It has no `prerequisites`
 - It has no `practices`
+
+### Updating the config
+
+Start by adding an entry to the `exercises.practice` array in the top-level `config.json` file.
+
+```
+{
+  "exercises": {
+    "practice": [
+      {
+        "uuid": "",
+        "slug": "hello-world",
+        "name": "Hello World",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
+      }
+    ]
+  }
+}
+```
+
+You can use the [Configlet][configlet] tool to get a UUID.
+Download Configlet by running `bin/fetch-configlet` from the root of the repository.
+Then generate a UUID using the `bin/configlet uuid` command.
+
+### Generating required files
+
+To implement the "Hello, World!" exercise, you'll need to create several files.
+Which files exactly is described in the [Practice Exercises documentation](/docs/building/tracks/practice-exercises).
+
+Most of the files can be added automatically by running Configlet's `sync` command:
+
+```
+bin/configlet sync --update --docs --metadata --tests
```

Comment:
```suggestion
bin/configlet sync --update --yes --docs --metadata --exercise hello-world
bin/configlet sync --update --tests include --exercise hello-world
```

Unfortunately, you can't yet do this in a single command (looks like a bug).

Reply:
Ah, I was wondering about that, but hadn't gotten around to asking why it was this way.